### PR TITLE
Docker development environment

### PR DESCRIPTION
**Pull request by Rukomoynikov**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    command: ["milvus", "run", "standalone"]
+    security_opt:
+      - seccomp:unconfined
+    environment:
+      ETCD_ENDPOINTS: etcd:2379
+      MINIO_ADDRESS: minio:9000
+    volumes:
+      - ${DOCKER_VOLUME_DIRECTORY:-.}/volumes/milvus:/var/lib/milvus
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9091/healthz"]
```
Comment (andreibondarev) [inline docker-compose.yml:60]: `http://localhost:9091/healthz`

Is this a typo: `z` at the end of the url?

**Review by andreibondarev — APPROVED** [review]
@Rukomoynikov Just 1 comment and it's good to go 🚀

File: docker-compose.yml (hunk 1 above)
Comment (Rukomoynikov) [reply docker-compose.yml:60]: > `http://localhost:9091/healthz`
> 
> Is this a typo: `z` at the end of the url?

Good question. 

I took the file directly from this Milvius page https://milvus.io/docs/install_standalone-docker-compose.md. Look at the file they suggest to use:

```bash
wget https://github.com/milvus-io/milvus/releases/download/v2.4.6/milvus-standalone-docker-compose.yml -O docker-compose.yml
```

Comment (andreibondarev) [reply docker-compose.yml:60]: If that's what's in there -- then we're good!

**Pull request merged**